data/udisks2.service.in

```diff
@@ -3,10 +3,27 @@ Description=Disk Manager
 Documentation=man:udisks(8)
 
 [Service]
-Type=dbus
 BusName=org.freedesktop.UDisks2
+CapabilityBoundingSet=CAP_SYS_RAWIO
```

Comment:
This should be a blacklist rather than whitelist.

Reply:
OK, how about:
`CapabilityBoundingSet=~CAP_DAC_OVERRIDE CAP_DAC_READ_SEARCH CAP_FOWNER CAP_FSETID CAP_KILL CAP_SETGID CAP_SETUID CAP_SETPCAP CAP_LINUX_IMMUTABLE CAP_NET_ADMIN CAP_NET_RAW CAP_IPC_LOCK CAP_IPC_OWNER CAP_SYS_CHROOT CAP_SYS_PTRACE CAP_SYS_PACCT CAP_SYS_BOOT CAP_SYS_NICE CAP_SYS_RESOURCE CAP_SYS_TIME CAP_SYS_TTY_CONFIG CAP_LEASE CAP_AUDIT_WRITE CAP_AUDIT_CONTROL CAP_SETFCAP CAP_MAC_OVERRIDE CAP_MAC_ADMIN CAP_SYSLOG CAP_WAKE_ALARM CAP_BLOCK_SUSPEND CAP_AUDIT_READ`
?

Reply:
`CAP_FOWNER` -- we use chmod to set permissions on mountpoints during mount
`CAP_SETGID` and `CAP_SETUID` -- we need to be able to run mount and unmount functions with different uids
`CAP_IPC_LOCK` -- this blocks `mlock` which is used by cryptsetup
`CAP_BLOCK_SUSPEND` -- we are using system inhibit to block suspend during long tasks, not sure if this prevents it or not